core/src/test/java/org/elasticsearch/index/mapper/core/NumberFieldTypeTests.java

```diff
@@ -21,8 +21,18 @@
 
 import com.carrotsearch.randomizedtesting.generators.RandomPicks;
 
```

Comment:
Nothing to do with this PR, but I don't like that we have this `NumberFieldTypeTests` which randomly chooses a type to test. We should be testing the mapper/fieldtype for every numeric type on every test run...